com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ApplicationMetadataBuilder.java

```diff
@@ -17,21 +17,32 @@
 import com.sap.cloud.lm.sl.mta.model.Module;
 import com.sap.cloud.lm.sl.mta.model.ProvidedDependency;
 
+import io.micrometer.core.instrument.util.StringUtils;
```

Comment:
I think it will be better to use StringUtils from apache commons (org.apache.commons.lang3)

Reply:
this was just a typo on my part, good catch!